Core/include/Acts/TrackFinding/CombinatorialKalmanFilter.hpp

```diff
@@ -1427,16 +1434,20 @@ class CombinatorialKalmanFilter {
                  << combKalmanResult.result.error().message()
                  << " with the initial parameters: \n"
                  << initialParameters.parameters());
-      return combKalmanResult.result.error();
     }
 
     std::vector<typename TrackContainer::TrackProxy> tracks;
 
     for (auto tip : combKalmanResult.lastMeasurementIndices) {
+      auto it = combKalmanResult.fittedParameters.find(tip);
+      if (it == combKalmanResult.fittedParameters.end()) {
+        continue;
```

Comment:
Should this be reported anyway? Previously any error would set the global error variable in the CKF result. It's now not used to fail the main `findTracks` call anymore, so is it useful at all? It also gets overwritten if a subsequent step produces an error as well, no?

Reply:
it would be good to report the error somehow through the track container or the result but I do not see a clear path towards this right now. do you think it makes sense to remove the result from the state for now?

Reply:
No strong opinion here.